Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1798,14 +1800,347 @@ def remove_nulls_from_dictionary(dict):
             del dict[key]
 
 
-def get_demisto_version():
+def assign_params(**kwargs):
+    """Creates a dictionary from given kwargs without empty values
```

Comment:
we might want to include '' sometimes, or other blank params, maybe you need to add an argument for skipped values?

Reply:
'', None, [], {} will be caught.